include/sys/socket.h

```diff
@@ -76,46 +76,46 @@ extern "C" {
 #endif
 
 
-typedef uint16_t sa_family_t;	// match lwIP size: u8 len + u8 family
+/* Match lwIP size: u8 len + u8 family. Align to allow safe sockaddr_* casting */
```

Comment:
Once added, now removed [`__attribute__(aligned(4))`](https://github.com/phoenix-rtos/libphoenix/commit/41d44ee02ac230c4578d928f76fb094d55cf8fdd#diff-34ef7636426053b9b87fd53d7eeb2100d372858fd0b755a4191ce5bcb2efb191R79-R80). Comment not updated.
```suggestion
/* Match lwIP size: u8 len + u8 family. */
```